README.md

```diff
@@ -29,6 +29,18 @@ WEBHOOK_PROXY_URL=<web hook url (like https://smee.io), not required>
 
 Add `BASE_BRANCH=master` or whatever is appropriate.
 
+#### Allowed users
+
+It is possible to restrict who can execute a benchmark. 
+
+Add `ALLOWED_USERS` with comma separated list of user's github ids. Eg: 
+
+`ALLOWED_USERS=123,455,234`
```

Comment:
can we not support usernames directly?

Reply:
I believe it would be possible. One of the reasons  that i went with ids was that you can your username ( although probably not very common thing to do).

Reply:
I think even if users can change their username, we would prefer this, as it is much easier to audit.

Reply:
Github IDs are immutable, therefore saner to rely on. You don't risk having someone hijack a username to execute the commands. I don't get why it would be harder to audit - even if that were the case, the stability makes up for it.